src/mbgl/tile/geometry_tile.hpp

```diff
@@ -59,7 +59,7 @@ class GeometryTile : public Tile {
 
     class PlacementResult {
     public:
-        std::unordered_map<std::string, std::shared_ptr<Bucket>> buckets;
+        std::unordered_map<std::string, std::shared_ptr<Bucket>> symbolBuckets;
```

Comment:
We could, however it would make `GeometryTile::getBucket` longer because it couldn't use `layer.is<SymbolLayer>() ? symbolBuckets : nonSymbolBuckets`. Not sure if it's worth it.